### ISSUE TITLE
tracking the trained image is still the issues

### ISSUE THREAD

**Issue by BishalPandeycsb** [Vrushank796/Face-recognition-and-attendance-system#1]
whenever i start to track the image it start printing unknown

**Comment by Vrushank796** [comment]
Can you share me the screenshot

**Comment by BishalPandeycsb** [comment]
these are the screenshot the csv file is being created but not the data
items are filled there

**Comment by Vrushank796** [comment]
I couldn’t see any ss

**Comment by BishalPandeycsb** [comment]
https://drive.google.com/open?id=10fkwaT47TecQVuRYFB4y1LkLivgjyelQ

**Comment by Vrushank796** [comment]
Bro,Just check if your name and other details are adding in excel sheet  after  submitting all details while filling up the form in API ,and
then check the logs in pycharm console while tracking images,also try it with another person details and identity

**Comment by annu0520** [comment]
Hey i am facing error in keyerror: Id 
![424C23A2-84E1-46A8-B9D3-D1C0D1EFC22C](https://user-images.githubusercontent.com/62607153/82141219-fb0e1b00-9851-11ea-9dee-1fcb4c154872.jpeg)

**Comment by Vrushank796** [comment]
Make sure you have tkinter installed and checkout excel file for your name ,if it is being automatically created or not with date and time,bcz it is showing keyvalue error while tracking images,right ,at the time of tracking images we are calling the value of our name that is entered in excel file while training and submitting details

**Comment by annu0520** [comment]
Tkinter is properly install and no excel file is created in attendance folder .

**Comment by Vrushank796** [comment]
First just check that if in student Details folder csv file is created or not

**Comment by annu0520** [comment]
Yes that folder is created and information is getting stored there . But track part is not working and attendance is not getting record.

**Comment by Vrushank796** [comment]
Can you send me ss of the error while tracking image

**Comment by swetabhmukherjee** [comment]
Its showing KeyError: 'Id' . How can this be fixed?

**Comment by BishalPandeycsb** [comment]
recent:///55a5771c7279d76111eef8c65eed6a9a
recent:///8c4af1dd4479b9834bbc80605eed6a9a
recent:///d4c00b73e040592e99dc99dc5eed6a9a

**Comment by BishalPandeycsb** [comment]
![Screenshot from 2020-06-20 07-13-29](https://user-images.githubusercontent.com/47141238/85188717-6a01e800-b2c6-11ea-9d17-7d325963fbd8.png)
still the details are not printed and not being saved on attendance.csv folder

**Comment by Vrushank796** [comment]
Are you training images

**Comment by Vrushank796** [comment]
> Its showing KeyError: 'Id' . How can this be fixed?

Please send me the ss of error
